packages/rules-unit-testing/src/impl/test_environment.ts

```diff
@@ -179,6 +202,7 @@ class RulesTestContextImpl implements RulesTestContext {
   }
   storage(bucketUrl?: string): firebase.storage.Storage {
```

Comment:
```suggestion
  storage(bucketUrl = `gs://${this.projectId}`): firebase.storage.Storage {
```

(I double checked this works)

Reply:
Done